usaspending_api/references/models.py

```diff
@@ -482,8 +482,8 @@ class Meta:
         unique_together = (('program_activity_code', 'budget_year', 'responsible_agency_id', 'allocation_transfer_agency_id', 'main_account_code'),)
 
 
-class CFDAProgram(DataSourceTrackedModel):
-    program_number = models.TextField(primary_key=True, max_length=7)
+class CFDA(DataSourceTrackedModel):
```

Comment:
I'm OK either way. Hopefully it doesn't cause any migration funkyness to change it. But just let me know if you want to pull the trigger on that and I can change it.